File: FrontEnd/src/app/register/userInfo.ts
export class UserInfo {
    username:string|null;
    password:string|null;
    mail:string|null;

    constructor(username:string|null, password:string|null, mail:string|null){
        this.username = username;
        this.password = password;
        this.mail = mail;
    }
}

File: usuarios_service/src/main/java/upm/tfg/b190222/usuarios_service/info/UserInfo.java
package upm.tfg.b190222.usuarios_service.info;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class UserInfo {
    private String username;
    private String password;
    private String mail;
}
